### PR TITLE
work on the "future" episode

### PR DESCRIPTION
**Pull request by bast**
I have removed a lot of text and tried to make this fit into 20 minutes.

### CODE REVIEW

File: _episodes/07-future.md [hunk 1]
```diff
@@ -140,178 +67,37 @@ Several avenues should be pursued:
...
->
-> - What do you think will be needed to reach sustainability?
-> - How do you think the lesson material should be maintained after
->   project funding ends?
-> - What type of partnerships do you think are possible?
-{: .task}
+- Become a workshop helper
+- Become a CodeRefinery workshop instructor
+- Become a Carpentries workshop instructor
+- Contribute to teaching material: <https://github.com/coderefinery>
```
Comment (rkdarst) [inline _episodes/07-future.md:77]:
```suggestion
- Contribute to teaching material: <https://github.com/coderefinery>
- Create your own teaching material in our constellation of teaching.
```

File: _episodes/07-future.md [hunk 2]
```diff
@@ -140,178 +67,37 @@ Several avenues should be pursued:
...
 
-> ## Existing organizations
->
-> What existing organizations in your locations are working on similar
-> problems?
-{: .challenge}
+- Attending an instructor training (this workshop).
+- Contribution to a lesson, either via pull request, issue, discussion, or as a helper.
+- Sharing a short teaching demo and discussing feedback or teaching a lesson at a CodeRefinery workshop.
+- Participating in a call with other instructors and helpers either preparing a workshop or during a workshop debrief session.
```
Comment (rkdarst) [inline _episodes/07-future.md:103]:
```suggestion
- Participating in a call with other instructors and helpers either preparing a workshop or during a workshop debrief session.
- There are many ways to learn how to teach well.  We and carpentries provide one way and basic material,
  but we want to find a way for everyone to be a part of us.
```
I'm not sure if saying this is a good idea, and I don't like the wording.  The feeling I am trying to give is it's not just that we want to make one official path to be a teacher (but we have a lot of really important stuff to say, that everyone should at least read).

File: _episodes/07-future.md [hunk 3]
```diff
@@ -9,118 +9,45 @@ questions:
...
+including project directive and project plan, are available on the
+[NeIC website](https://neic.no/coderefinery/).
 
----
 
-## Vision
+### Vision: institutional partnerships
 
 - After phase 2, travel and accommodation for workshops will be financed by the hosting
   institution, similarly to the Carpentries model.
```
[the remark this answers is not in the document]
Comment (bast) [reply _episodes/07-future.md:39]: It often starts by paying outside staff but in the longer term hopefully also internal staff takes instructor training.

**Comment by bast** [comment]
Sorry I deleted a lot of your text. I can bring it back. I find HoSC important and it deserves a mention. Should I try to condense the previous text?

**Comment by bast** [comment]
Indeed I am convinced now that HoSC and the bigger picture should there be, too. I am happy to reintroduce unless you have a more uptodate text without too much work for you.

**Comment by bast** [comment]
> HoSC and the bigger picture should there be

Followed up in #110.

**Pull request merged**